src/jj2lsj.jl

```diff
@@ -99,11 +99,11 @@ function jj2lsj(::Type{T}, orbs::RelativisticOrbital...) where T
         i = findall(isequal((n,ℓ)), nℓs)
         subspace = orbs[i]
         mⱼ = map(subspace) do orb
-            j = orb.j
+            j = convert(Rational, orb.j)
             -j:j
         end
 
-        jₘₐₓ = maximum([o.j for o in subspace])
+        jₘₐₓ = maximum([convert(Rational, o.j) for o in subspace])
```

Comment:
I didn't convert this file over yet, hence the `convert`s. In the long term, would it make sense to enforce `A,B,C <: Union{Integer,HalfInteger}` for the `ClebschGordan` type parameter?